product_barcode_constraint_per_company/__manifest__.py

```diff
@@ -6,13 +6,13 @@
     "name": "Product Barcode Constraint per Company",
     "version": "14.0.1.0.1",
     "category": "Product",
-    "summary": "Change the product barcode constraint"
-    ", allowing the same barcode for differents companies",
+    "summary": """Change the product barcode constraint,
+                  allowing the same barcode for differents companies""",
     "author": "GRAP, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/stock-logistics-barcode",
     "license": "AGPL-3",
     "depends": [
-        "product",
+        "product_variant_company",
```

Comment:
You mustn't change dependencies in a stable module.

Reply:
Hi @pedrobaeza , AFAICS this is not a stable module, and the change in dependency was asked here: https://github.com/OCA/stock-logistics-barcode/pull/526#pullrequestreview-1658253604 following https://github.com/OCA/stock-logistics-barcode/pull/526#discussion_r1275978467

Reply:
OK then about this.